jenkins-vars.yml

```diff
@@ -20,7 +20,7 @@ repo_vars:
   - CI_WEB = 'true'
   - CI_PORT = '8080'
   - CI_SSL = 'false'
-  - CI_DOCKERENV = 'TEST_RUN=true|DB_HOSTNAME=localhost|DB_USERNAME=postgres|DB_PASSWORD=password|DB_DATABASE_NAME=postgres|REDIS_HOSTNAME=localhost'
+  - CI_DOCKERENV = 'DB_HOSTNAME=localhost|DB_USERNAME=postgres|DB_PASSWORD=postgres|DB_DATABASE_NAME=postgres|REDIS_HOSTNAME=localhost|DOCKER_MODS=imagegenius/mods:universal-postgres|imagegenius/mods:universal-redis'
```

Comment:
`DOCKER_MODS=imagegenius/mods:universal-postgres|imagegenius/mods:universal-redis`

`imagegenius/mods:universal-redis` is being pharsed as another variable